spec/models/book_index_state_spec.rb

```diff
@@ -14,12 +14,17 @@
   subject(:book_index_state) { described_class }
 
   describe ".create" do
-    it 'creates the a document row in the dynamo db table' do
+    it 'creates the a document row in the dynamo db table with correct status' do
```

Comment:
`row` => `item` -- so we don't confuse future developers into thinking dynamo is a RDBMS :-)